### ISSUE TITLE
Exception when the camera returns no image

### ISSUE THREAD

**Issue by jfisteus** [jfisteus/eyegrade#114]
This exception happens in the camera selector dialog. It might also happen while grading.

```
OpenCV Error: Assertion failed (ssize.area() > 0) in resize, file -------src-dir-------/opencv-2.4.10/modules/imgproc/src/imgwarp.cpp, line 1968
Traceback (most recent call last):
  File "eyegrade/eyegrade/qtgui/dialogs.py", line 355, in _next_capture
    image = self.capture_context.capture(resize=(320, 240))
  File "eyegrade/eyegrade/detection.py", line 527, in capture
    image = cv2.resize(image, resize, interpolation=cv2.INTER_AREA)
cv2.error: -------src-dir-------/opencv-2.4.10/modules/imgproc/src/imgwarp.cpp:1968: error: (-215) ssize.area() > 0 in function resize
```

**Comment by jfisteus** [comment]
A related exception that happens in a similar situation:

```
OpenCV Error: Assertion failed (ssize.area() > 0) in resize, file /build/opencv-FWWjHr/opencv-2.4.9.1+dfsg/modules/imgproc/src/imgwarp.cpp, line 1834
Traceback (most recent call last):
  File "eyegrade/qtgui/dialogs.py", line 355, in _next_capture
  File "eyegrade/detection.py", line 527, in capture
cv2.error: /build/opencv-FWWjHr/opencv-2.4.9.1+dfsg/modules/imgproc/src/imgwarp.cpp:1834: error: (-215) ssize.area() > 0 in function resize
```

**Comment by jfisteus** [comment]
Closed for the next release (Eyegrade 0.7)
